tests/suites/test_suite_x509write.data

```diff
@@ -265,3 +265,9 @@ mbedtls_x509_string_to_names:"C=NL, 2.5.4.10.234.532=#0C084F6666737061726B, OU=P
 
 Check max serial length
 x509_set_serial_check:
+
+Check max extension length (Max-1)
+x509_set_extension_length_check:0xFFFFFFFE
```

Comment:
In theory this should succeed on any platform that has enough memory for it. I think it's fine to remove this case, since it is not technically an error case.